roles/hive/defaults/main.yaml

```diff
@@ -0,0 +1,165 @@
+# Used to cleanup the hive installation
+hive_cleanup: false
+
+hive_repo_address: "https://github.com/ethereum/hive.git"
+hive_repo_version: "master"
+hive_repo_remote: "origin"
+
+hive_systemd_service_name: "hive.service"
+
+hive_dir: "/data/hive"
+hive_dir_bin: "{{ hive_dir }}/bin"
+hive_dir_source: "{{ hive_dir }}/src"
+hive_dir_scripts: "{{ hive_dir }}/scripts"
+hive_dir_results: "{{ hive_dir }}/results"
+hive_dir_web: "{{ hive_dir }}/web"
+
+hive_container_www_name: "nginx-hive"
+hive_container_www_image: "nginx:1.27.3-alpine"
+hive_container_www_env: {}
+hive_container_www_ports:
+  - "127.0.0.1:8080:80"
+hive_container_www_volumes:
+  - "{{ hive_dir_web }}:/usr/share/nginx/html"
+  - "{{ hive_dir_results }}:/usr/share/nginx/html/results"
+hive_container_www_networks: []
+hive_container_www_restart_policy: "always"
+hive_container_www_stop_timeout: 10
+
+
+hive_simulations:
+  # Always pull / build the latest container images
+  always_pull_images: true
+  # Global flags provided to all test runs
+  global_flags:
+    - --docker.nocache=.
```

Comment:
could we make loglevel configurable? just {{ log_level }} and then log_level: 3 or similar

Reply:
It is already configurable. You just need to change the flag value. Do you mean, having it's own variable for it?

Reply:
Added vars for log levels 👍